### PR TITLE
CSS fixes

### PR DESCRIPTION
**Pull request by OmeGak**
Depends on https://github.com/indico/indico/pull/2219

### CODE REVIEW

File: indico/htdocs/js/indico/jquery/clearableinput.js [hunk 1]
```diff
@@ -31,15 +31,16 @@
         _create: function() {
             var self = this;
 
-            self.buttonBox = $('<span class="button-box"></span>');
-            self.clearIcon = $('<a class="input-clear icon-close"></a>')
+            self.buttonBox = $('<span>').addClass('button-box');
```
Comment (nop33) [inline indico/htdocs/js/indico/jquery/clearableinput.js]: Personally I prefer the syntax
``` js
self.buttonBox = $('<span>', {
    class: 'button-box'
});
```
which is also proven to be a bit faster. But in this case, your way is more readable, so I don't have a strong opinion.

File: indico/htdocs/sass/partials/_inputs.scss [hunk 2]
```diff
@@ -371,9 +383,10 @@ $dropdown-transition: $dropdown-transition-step ease-out;
         display: none;
         opacity: 0;
     }
-    > label.i-synced-label {
-        text-align: center;
-        line-height: 2.3em;
+
+    > label.i-synced-label:before {
+        // Fixes label vertical-align
```
Comment (nop33) [inline indico/htdocs/sass/partials/_inputs.scss]: :+1:

File: indico/htdocs/sass/partials/_forms.scss [hunk 3]
```diff
@@ -30,380 +50,270 @@
...
-            float: left;
-            text-align: right;
-            width: 20%;
+        .handle {
+            @extend %form-field-input;
+            border-right: 0;
+            border-top-right-radius: 0;
+            border-bottom-right-radius: 0;
+            padding: 0;
+            line-height: 2.5em;
```
Comment (nop33) [inline indico/htdocs/sass/partials/_forms.scss]: I guess now it's not the time to start converting to unit-less, right?

File: indico/htdocs/js/indico/jquery/clearableinput.js (hunk 1 above)
Comment (OmeGak) [inline indico/htdocs/js/indico/jquery/clearableinput.js]: Me neither. Feel free to change it in the next round of improvements. Gotta move on.

File: indico/htdocs/sass/partials/_forms.scss (hunk 3 above)
Comment (OmeGak) [inline indico/htdocs/sass/partials/_forms.scss]: Yeah... Let's keep it for next round of CSS cleanup.

**Pull request merged**